### PR TITLE
Config docs update with note for AWS Lambda

### LINKED ISSUE #1704: ElasticAPM is hanging lambda function after processing all events in an SQS queue
I believe ElasticAPM will hang the lambda function when no more events exist in the SQS queue.

**To Reproduce**

* Setup a lambda function to process events from an SQS queue
* Send events to an SQS queue
* Set `ELASTIC_APM_LOG_LEVEL: debug`

**Environment (please complete the following information)**
- OS:  debian:bullseye-slim (python:3.9-slim container image) x86
- Python version: 3.9
- Framework and version [e.g. Django 2.1]:
- APM Server version: 8.4.2
- Agent version: 6.13.2

**Additional context**
Cloudwatch logs (can dive deeper if needed) after execution is completed
image

### PR DESCRIPTION
**Pull request by brett-fitz**
## What does this pull request do?

Updated docs with a note about aws lambda configuration, see GH-1704 for more info.

## Related issues

Closes #1704

### CODE REVIEW

**Comment by brett-fitz** [comment]
Signed contributor agreement.

File: docs/configuration.asciidoc [hunk 1]
```diff
@@ -115,6 +115,7 @@ or whitespace-only.
 
 The URL for your APM Server.
 The URL must be fully qualified, including protocol (`http` or `https`) and port.
+Note: Do not set this if you are using APM in an AWS lambda function. APM Agents are designed to proxy their calls to the APM Server through the lambda extension.
```
Comment (basepi) [inline docs/configuration.asciidoc:118]:
```suggestion
Note: Do not set this if you are using APM in an AWS lambda function. APM Agents are designed to proxy their calls to the APM Server through the lambda extension. Instead, set `ELASTIC_APM_LAMBDA_APM_SERVER`. For more info, see <<lambda-support,AWS Lambda>>.
```

**Review by basepi — COMMENTED** [review]
Looks great! What do you think about calling out the correct config value instead? I also added a link to the lambda docs.

File: docs/configuration.asciidoc (hunk 1 above)
Comment (brett-fitz) [reply docs/configuration.asciidoc]: Great idea!

**Review by basepi — APPROVED** [review]

**Comment by basepi** [comment]
run elasticsearch-ci/docs

**Comment by basepi** [comment]
@brett-fitz I think the CLA checker is mad because you have a typo in the email you used for your first commit: `brett.fitzpatick@ironnetcybersecurity.com` (note the missing `r` in `fitzpatrick`)

Either sign for that email or you may have to rebase your commits after fixing your git config.

**Comment by brett-fitz** [comment]
Apologies for the delay as I was OOO. I will simply sign the CLA with the typo'd email address. For future commits this should be fixed. Thank you for pointing this out!

**Comment by brett-fitz** [comment]
Just attempted again, let me know if that resolved it. Otherwise I can dump this one and open a new PR.

**Pull request merged**